docs/website/Makefile

```diff
@@ -10,4 +12,7 @@ all:
 	cp mtouch-errors.md $(WEBSITE_DOCS)/guides/ios/troubleshooting/mtouch-errors/index.md
 	cp binding_objc_libs.md $(WEBSITE_DOCS)/guides/ios/advanced_topics/binding_objective-c/binding_objc_libs/index.md
 	cp binding_types_reference_guide.md $(WEBSITE_DOCS)/guides/ios/advanced_topics/binding_objective-c/binding_types_reference_guide/index.md 
-	
+
+xamarin-analysis-doc:
```

Comment:
that should be added as a dependency to `all`
and that needs a _final_ location on the web site for the document